app/decorators/ncr/work_order_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Ncr
+  class WorkOrderDecorator < Draper::Decorator
+    delegate_all
+
+    EMERGENCY_APPROVER_EMAIL = 'Emergency - Verbal Approval'
+    NO_APPROVER_FOUND        = 'No Approver Found'
```

Comment:
I don't think we are doing `=` alignment in the rest of this project, so I would vote to not do it here. I don't feel strongly about this "rule" per se but I do feel strongly about consistency within a project!